manifests/ruby.yml

```diff
@@ -631,9 +631,7 @@ tests/:
     Test_Span_Links_From_Conflicting_Contexts: missing_feature
     Test_Span_Links_Omit_Tracestate_From_Conflicting_Contexts: missing_feature (implementation specs have not been determined)
   test_graphql.py:
-    Test_GraphQLQueryErrorReporting:
```

Comment:
That change seems independent from the PR description. Is that meant to be in here? :)

Reply:
Yes, the test `tests/test_graphql.py` was changed in this PR, and depends on v0.4 array serialization, which was changed for this PR.